src/transform/DimImagesTransform.js

```diff
@@ -0,0 +1,25 @@
+import './DimImagesTransform.css'
+const CLASS = 'pagelib-dim-images'
+
+/**
+ * @param {!Window} window
+ * @param {!boolean} shouldDim
+ * @return {void}
+ */
+const dimImages = (window, shouldDim) => {
+  window.document.querySelector('html')
+    .classList[shouldDim ? 'add' : 'remove'](CLASS)
```

Comment:
Feels like a very Javascript-y thing... which gives me pause, but it's just so concise :)